src/main/java/org/terasology/climateConditions/FrostbiteSystem.java

```diff
@@ -36,6 +36,10 @@
 import org.terasology.utilities.random.FastRandom;
 import org.terasology.utilities.random.Random;
 
+/**
+ * This system is responsible for periodic frostbite damage (along with stun and a sound effect) to the player as long
+ * as the player has a Hypothermia component.
+ */
```

Comment:
```suggestion
/**
 * Adds frostbite to the player.
 * Frostbite is a periodic effect that, e.g., damages and stuns the player.
 * Is only active iff the player has a {@link HypothermiaComponent}.
 */
```

With "iff" being "if and only if".